hero:   
        hero_robot:
                type: hero_chassis_controller/HeroChassisController
                left1_joint: left_front_wheel_joint
                left2_joint: left_back_wheel_joint
                right1_joint: right_front_wheel_joint
                right2_joint: right_back_wheel_joint
                pid1: {p: 1000, i: 0.001, d: 0}
                pid2: {p: 1000, i: 0.001, d: 0}
                pid3: {p: 1000, i: 0.001, d: 0}
                pid4: {p: 1000, i: 0.001, d: 0}
                wheelBase: 0.4
                trackWidth: 0.4
                radius: 0.07

                

